src/ert/analysis/_es_update.py

```diff
@@ -464,15 +446,23 @@ def _update_with_row_scaling(
         temp_storage = _create_temporary_parameter_storage(
             source, iens_active_index, param_group.name
         )
+
+        # Define inversion map from ERT to ies api
+        inversion_types = {0: "exact", 1: "subspace", 2: "subspace", 3: "subspace"}
```

Comment:
Perhaps we could pass this mapping into the function so that we don't have to define it twice in the same file?

Reply:
I suggest adding it to: `IESSettings` in `analysis_module.py`, then we can have all the settings in one place.

Reply:
I am unsure about how I feel with regards to how parameters are contained in a settings object which is passed around to every function. Perhaps more clean according to some measure? I am not surprised if this is some kind of pattern, at least in ERT. The code feels however harder to read for me when we do this, so I have to meditate on the trade-off here. I usually find functions and classes that takes only the arguments they need easier to read and understand. What do you think?

Reply:
So I think I prefer @dafeda proposal. But I am not an expert here, so feel free to tell me